main.py

```
@@ -6,37 +6,26 @@
+def process_box_score(event: CloudEvent):
     storage_client = storage.Client()
-    bucket = storage_client.get_bucket(event['bucket'])
-    blob = bucket.get_blob(event['name'])
-    print('Event ID: %s bucket: %s object: %s' % (context.event_id, bucket, blob))
-    if event['name'].find('-replay') > 0:
+    bucket_name = event.data['bucket']
+    bucket = storage_client.get_bucket(bucket_name)
+    blob_name = event.data['name']
+    blob = bucket.get_blob(blob_name)
+    if blob is None:
+        raise Exception(f"Object not found gs://{bucket_name}/{blob_name}")
```

Ruff (0.11.9)

Create your own exception

(TRY002)